packages/integration/config.d.ts

```diff
@@ -62,6 +62,11 @@ export interface Config {
         tenantId?: string;
         personalAccessToken?: string;
       }[];
+      /**
+       * PGP signing key for signing commits.
+       * @visibility secret
+       */
+      signingKey?: string;
```

Comment:
wonder if we should rename this to `commitSigningKey` instead? The PGP key isn't used for anything else right?

Reply:
Renamed to `commitSigningKey` and `defaultCommitSigningKey` accordingly